app/app.ts

```diff
@@ -27,10 +26,18 @@ import ChildWindow from 'components/windows/ChildWindow.vue';
 import OneOffWindow from 'components/windows/OneOffWindow.vue';
 
 const { ipcRenderer, remote } = electron;
-
 const slobsVersion = remote.process.env.SLOBS_VERSION;
 const isProduction = process.env.NODE_ENV === 'production';
 
+window['obs'] = window['require']('obs-studio-node');
+
+{ // Set up things for IPC
+  // Connect to the IPC Server
+  window['obs'].IPC.connect(remote.process.env.SLOBS_IPC_PATH);
+  document.addEventListener('close', (e) => {
+    window['obs'].IPC.disconnect();
+  });
+}
```

Comment:
Honestly I've never seen someone use a block like this in javascript outside the context of the normal uses such as functions and conditionals.  I'm curious what the motivation here is.  Are you just grouping these statements together for the purpose of readability?